File: week4-api-microservices/javascript/Test03_SaveTask.js
const { client } = require("./connection");

console.log("========================================");
console.log("Start exercise");

(async () => {
  try {
    await client.execute(
      "INSERT INTO todoitems (user_id, item_id, completed, title, offset) VALUES ( 'john', 11111111-5cff-11ec-be16-1fedb0dfd057, true, 'Walk the dog', 0);"
    );

    console.log("SUCCESS");
  } catch (e) {
    console.log(e);
  }
  console.log("========================================");
  process.exit();
})();

File: week4-api-microservices/python/Test03_SaveTask.py
#!/usr/bin/env python3
import uuid
from connection import session

print('========================================')
print('Start exercise')
try:
    session.execute(
        "INSERT INTO todoitems (user_id, item_id, completed, title, offset) VALUES ( 'john', 11111111-5cff-11ec-be16-1fedb0dfd057, true, 'Walk the dog', 0);")
except Exception as e:
    print(e)
    print('Failure')
else:
    print('Success')
print('========================================')
